src/types.ts

```diff
@@ -211,3 +211,25 @@ export type AppClient = {
    */
   requirePkce: boolean;
 };
+
+export type UserInfo = {
+  sub: string;
+  email?: string;
+  email_verified?: boolean;
+  name: string;
+  website?: string;
+  zoneinfo?: string;
+  given_name?: string;
+  middle_name?: string;
+  family_name?: string;
+  preferred_username?: string;
+  phone_number?: string;
+  phone_number_verified?: boolean;
+  locale?: string;
+  updated_at: number;
+  picture?: string;
+  address?: string;
+  birthdate?: string;
+  created_at: number;
+  modified_at: number;
```

Comment:
This belongs back in `src/oidc/format/json.ts`.

We do need our own UserInfo object, but it uses camelCase like all other types in this file, and should only have properties for the specific things that you added to the database schema.

Reply:
Ohhh I think I'm starting to see the difference between when _ and camelcase is used...